send-transaction-service/src/send_transaction_service.rs

```diff
@@ -566,12 +568,18 @@ impl SendTransactionService {
         stats: &SendTransactionServiceStats,
     ) {
         // Processing the transactions in batch
-        let addresses = Self::get_tpu_addresses_with_slots(
+        let mut addresses = config
+            .tpu_peers
+            .as_ref()
+            .map(|addrs| addrs.iter().map(|a| (a, 0)).collect::<Vec<_>>())
```

Comment:
This line would be so much simpler and easier to read if tpu_peers was just a Vec.